README.md

```diff
@@ -319,7 +319,22 @@ We would be happy to merge your feature then.
 With this command, you can process your project with rector from docker:
 
 ```bash
-docker run -v $(pwd):/project rector/rector:latest process project/app --level symfony40 --dry-run
+docker run -v $(pwd):/project rector/rector:latest process project/src --level symfony40 --dry-run
 
 # Note that a volume is mounted from `pwd` into `/project` which can be accessed later.
 ```
+
+Using `rector.yaml`:
+
+```bash
+docker run --rm -v $(pwd):/project -t rector process project/src --config project/rector.yaml --dry-run
+```
+
+```yaml
+parameters:
+    autoload_paths:
+        - 'project/vendor/autoload.php'
```

Comment:
Imo it is better practice to not bother about docker in your config, that this should be passed via CLI parameters when running via Docker. I am not sure if rector will fail or not if your colleague will try to run rector without Docker and rector will not be able to find this autoload path - does it fail or not?

Anyways rector will not work when omitting this completely, right?

Reply:
This is tricky. Better not to specify anything and use a cli flag.